frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.pdf.component.js

```diff
@@ -1,19 +1,611 @@
 import React from "react";
 import RenderPage from "../render-page.component.js";
 import PositionedString from "../pdf-rendering/positioned-string.component.js";
+import PositionedCheckmark from "../pdf-rendering/positioned-checkmark.component.js";
+import {
+  getJudicialDistrict,
+  getCounty
+} from "../form-common-options/form-common-options.js";
+import moment from "moment";
 
-export default class MotionToWaiveFees_Pdf extends React.Component {
-  render() {
-    const { renderData } = this.props;
-    return (
-      <>
-        <RenderPage url="/static/forms/motion-to-waive-fees/01_Motion_and_Affidavit_to_Waive_Fees-1.png">
-          <PositionedString debugKey="name" left="11%" top="14.04%">
-            {renderData("person.firstName")} {renderData("person.lastName")}
-          </PositionedString>
-        </RenderPage>
-        <RenderPage url="/static/forms/motion-to-waive-fees/01_Motion_and_Affidavit_to_Waive_Fees-2.png"></RenderPage>
-      </>
-    );
-  }
+export default function MotionToWaiveFees_Pdf({ data, renderData }) {
+  return (
+    <>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-01.png">
+        <PositionedString debugKey="fullName" left={farLeft} top="14.06%">
+          {`${renderData("person.firstName")} ${renderData(
+            "person.middleName"
+          )} ${renderData("person.lastName")}`}
+        </PositionedString>
+        <PositionedString
+          dataKey="person.addressStreet"
+          left={farLeft}
+          top="17.50%"
+        />
+        <PositionedString debugKey="lineAddress" left={farLeft} top="20.90%">
+          {`${renderData("person.addressCity")} ${renderData(
+            "person.addressState"
+          )} ${renderData("person.addressZip")}`}
+        </PositionedString>
+
+        <PositionedString dataKey="person.phone" left={farLeft} top="24.75%" />
+        <PositionedString dataKey="person.email" left={farLeft} top="27.90%" />
+        <PositionedCheckmark
+          dataKey="person.relation"
+          left="16.4%"
+          top="31.68%"
+          shouldShow={data.person.relation === "plantiffPetitioner"}
+        />
+        <PositionedCheckmark
+          dataKey="person.relation"
+          left="16.4%"
+          top="33.08%"
+          shouldShow={data.person.relation === "plantiffPetitionerAttorney"}
+        />
+        <PositionedCheckmark
+          dataKey="person.relation"
+          left="40.7%"
+          top="31.68%"
+          shouldShow={data.person.relation === "defendantRespondent"}
+        />
+        <PositionedCheckmark
+          dataKey="person.relation"
+          left="40.7%"
+          top="33.08%"
+          shouldShow={data.person.relation === "defendantRespondentAttorney"}
+        />
```

Comment:
since all of these are based off of `person.relation` i might try to dry it up a bit. that said; since the positioning is different in all of them it might not be worth it.